ui/config/webpack.config.prod.js

```diff
@@ -308,4 +306,7 @@ module.exports = {
     tls: 'empty',
     child_process: 'empty',
   },
+  performance: {
```

Comment:
What does this do?

Reply:
Turns off the "bundleSize" warning for prod builds (6MB > 250kb bundle). Any warning is interpreted as a failure in CI.